_summaries/srivastava2014dropout_1.md

```diff
@@ -1,19 +1,114 @@
+
+Dropout is a technique of randomly removing several nodes and their incoming and outgoing edges from the network. The removal probability of each node is independent from each other. We call the network post-removal as the "thinned" network. For each a training example, a new "thinned" network is trained, as the dropout is performed differently for every example. 
+
+One of the main motivations in the paper is adapting a network so that each unit can be trained to work with a random set of previous units as any unit can be randomly dropped. 
+
+![Diagram of Removal](srivastava2014dropout_1a.png)
+
+# Technical Implementation
+
+During testing and validation, we are unable to explicitly average the predictions from all thinned models. Instead, we use a single neural network without performing dropout, but scale the weights down with the dropout rate for each node. 
+
+![Diagram of Removal](srivastava2014dropout_1b.png)
+
+In terms of seeing a description of the model, we can see the following:
+
+![Diagram of Removal](srivastava2014dropout_1e.png)
+
+Here, we can treat $$y$$ as the output of a unit, $$z$$ as the input of a layer, $$f$$ as the activation function, $$w$$ as the weights, and $$b$$ as the biases. These values can be calculated using the following equations: 
+
+![Diagram of Removal](srivastava2014dropout_1c.png)
+
+In the second equation, we can treat $$r$$ as a vector of independent Bernoulli random variables which have a probability $$p$$ of being 1. This is multiplied across the outputs of the layer to thin the outputs, which are used as input into the next layer. 
+
+## Training Dropout Nets
+
+Training a dropout network using SGD is similar to a standard neural network. The main difference is that when performing forward and backpropagation, the gradients for each parameter are averaged over the cases in each batch. 
+
+
+# Performance
+
+The paper was tested on the following datasets:
+* MNIST: A set of handwritten digits for image classification
+* TIMIT: A speech benchmark for clean speech recongition
+* CIFAR-10 and CIFAR-100: Image classification sets
+* SVHN: Images of house numbers for classification to identify the digits
+* ImageNet: A collection of natural images for classification
+* Reuters-RCV1: A collection of Reuters articles
+* Alternative Splicing data set: RNA features for prediciting alternative gene splicing
+
+These sets were chosen to show that dropout is a general technique and can be used across multiple domains and tasks. 
+
+First, the paper examines the performance of dropout across the image data sets: MNIST, SVHN, CIFAR-10, CIFAR-100, and ImageNet. Across of the sets, state-of-the-art results were achieved. 
+
```

Comment:
The summary on experiments is extensive. My suggestion would be make it a bit conciser and include some key observation. For example, some things I notice interesting are: (1) applying dropout at all layers beat applying dropout at fc layers only as well as no dropout (question: what is common practice nowadays? is this conclusion still holds? if not, what is the reason behind?) (2) The improvement of using dropout on Reuters-RCV1 is less significant than on vision and speech task. (question: what could be some reasonable explanation?)